project/src/main/webapp/pages/course.html

```diff
@@ -9,6 +9,12 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link href="/styles/materialize.css" type="text/css" rel="stylesheet" media="screen,projection"/>
     <link href="/styles/style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
+
+    <!-- JS  -->
+    <script src="/scripts/createNewAssignment.js"></script>
+    <script src="/scripts/getCourseDetails.js"></script>
+    <!-- <script src="/scripts/listAssignments.js"></script> -->
```

Comment:
Remove unneeded code rather than commenting it out

Reply:
Thanks